### PR TITLE
games-action/minetest: version bump to 0.4.17

### PR DESCRIPTION
**Pull request by vilhelmgray**
This version adds support for the jsoncpp USE flag.

Closes: https://bugs.gentoo.org/657532

### CODE REVIEW

File: games-action/minetest/minetest-0.4.17.ebuild [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils eutils gnome2-utils user vcs-snapshot
```
Comment (Zlogene) [inline games-action/minetest/minetest-0.4.17.ebuild]: why do you need `vcs-snapshot` and `eutils`?

Comment (vilhelmgray) [reply games-action/minetest/minetest-0.4.17.ebuild]: `eutils` is required for the **usex** lines. `vcs-snapshot` is no longer necessary however so I'll remove it.

File: games-action/minetest/minetest-0.4.17.ebuild [hunk 2]
```diff
@@ -0,0 +1,136 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils eutils gnome2-utils user
```
Comment (Zlogene) [inline games-action/minetest/minetest-0.4.17.ebuild]: no, `usex` is the part of `EAPI >=5`

Comment (Zlogene) [reply games-action/minetest/minetest-0.4.17.ebuild]: https://devmanual.gentoo.org/ebuild-writing/eapi/index.html

Comment (vilhelmgray) [reply games-action/minetest/minetest-0.4.17.ebuild]: Ah okay, I'll remove eutils as well then.

**Pull request closed without merging**